upstream-community-operators/knative-serving-operator/0.11.0/knative-serving-operator.v0.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,212 @@
+    categories: 'Networking,Integration & Delivery,Cloud Provider,Developer Tools'
+    certified: 'false'
+    createdAt: '2020-01-14T00:00:00Z'
+    description: |-
+      Knative Serving builds on Kubernetes to support deploying and
+      serving of serverless applications and functions.
+    containerImage: 'gcr.io/knative-releases/knative.dev/serving-operator/cmd/manager:v0.11.1'
+    support: The Knative Authors
+    capabilities: Basic Install
+    repository: 'https://github.com/knative/serving-operator'
+spec:
+  displayName: Knative Serving Operator
+  description: >+
+    ## Knative Serving
+
+
+    Knative Serving builds on Kubernetes to support deploying and
+
+    serving of serverless applications and functions. The Knative
+
+    Serving project provides middleware primitives that enable:
+
+
+    - Rapid deployment of serverless containers
+
+    - Automatic scaling up and down to zero
+
+    - Routing and network programming for Istio components
+
+    - Point-in-time snapshots of deployed code and configurations
+
+    ## Further Information
+
+
+    For documentation on using Knative Serving, see the [Knative documentation
+    site](https://www.knative.dev/docs/serving/).
+
+
+
+  maturity: alpha
+  version: 0.11.0
+  replaces: ''
```

Comment:
please fill the value or remove whole line

Reply:
@J0zi done. thanks!